Music/quodlibet-control.1s.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# <bitbar.title>Quod Libet Control</bitbar.title>
+# <bitbar.version>v1.0</bitbar.version>
+# <bitbar.author>Sean Lane</bitbar.author>
+# <bitbar.author.github>seanlane</bitbar.author.github>
+# <bitbar.desc>Control Quoid Libet via Bitbar</bitbar.desc>
+# <bitbar.image>https://i.imgur.com/VA8ZEbR.png</bitbar.image>
+# <bitbar.dependencies>quodlibet, python</bitbar.dependencies>
+
+import os, shutil, subprocess, sys
```

Comment:
You are importing `shutil` but it's not used anywere.  Causing pyflakes to give you an linting error